portal/templates/redesign/teach_new/dashboard.html

```diff
@@ -268,21 +270,46 @@ <h4 class="col-sm-6 col-center">As an administrator of your school or club, you
         <h3>Two Factor Authentication</h3>
         <p class="body-text">Use your smartphone or tablet to enhance your account&rsquo;s security.</p>
-        <div class="row row--regular">
-            <div class="col-sm-5">
-                <h4>Backup tokens</h4>
-                <p class="body-text">If you don&rsquo;t have your smartphone or tablet with you, you can access your account using backup tokens. You have 10 backup tokens
-                    remaining.</p>
-                <p class="body-text">View and create backup tokens for your account.</p>
-                <button class="button--regular button--primary--negative">Manage backup tokens</button>
-            </div>
-            <div class="col-sm-6 col-sm-offset-1">
-                <h4>Disable Two Factor Authentication</h4>
-                <p class="body-text">We recommend you to continue using Two Factor Authentication, however you can disable two factor authentication for your account.</p>
-                <button class="button--regular button--secondary button--secondary--dark">Disable Two-Factor Authentication</button>
+
+        {% if user|has_2FA %}
+            <div class="row row--regular">
+                <div class="col-sm-5">
+                    <h4>Backup tokens</h4>
+                    <p class="body-text">If you don&rsquo;t have your smartphone or tablet with you, you can access your account using backup tokens.
+                        {% if backup_tokens == 1 %}
+                        You have only one backup token remaining.
+                        {% else %}
+                        You have {{ backup_tokens }} backup tokens remaining.
+                        {% endif %}</p>
+                    <p class="body-text">View and create backup tokens for your account.</p>
+                    <button class="button--regular button--primary--negative">Manage backup tokens</button>
+                </div>
+                <div class="col-sm-6 col-sm-offset-1">
+                    <h4>Disable Two Factor Authentication</h4>
+                    <p class="body-text">We recommend you to continue using Two Factor Authentication, however you can disable two factor authentication for your account.</p>
+                    <button class="button--regular button--secondary button--secondary--dark">Disable Two-Factor Authentication</button>
+                </div>
             </div>
-        </div>
+        {% else %}
+            <p class="body-text">Find out more about Two Factor Authentication and how it helps keep your account secure.</p>
+            <button class='button--regular button--primary--negative'>Setup Two Factor Authentication</button>
+        {% endif %}
     </div>
 </div>
 
+<script type="text/javascript">
+    $(window).scroll(function() {
+        var currentScroll = $(window).scrollTop();
+        if (currentScroll >= 343) {
+            $('.sub-nav--hidden').css({
+                'display': 'block'
+            });
```

Comment:
If your `.sub-nav--hidden` class is `display: block` it's no longer `hidden`.

When it's shown, change the class to `.sub-nav--shown` and put the display details in your CSS file.